### DISCUSSION TITLE
[Feature]: FAQ Page

### DISCUSSION THREAD

**Discussion by m4saurabh** [remix-run/remix#2609] [Proposals]
### Create a FAQ Page, and link it on the header.

Well, there a lot of common question which need answers, which are good to answer on the website itself and you refer people to the FAQ Page.

Some of the common questions are:
- Remix vs Next.js(though I know the answer of it)
- How it's going to monetized
- Future goals for Remix

**Comment by sergiodxa** [comment]
Another frequently asked question "Why Remix uses let instead of const?" (I'm not joking, if there's a FAQ it should be there)

**Reply by m4saurabh** [reply]
I know the answer of it,
https://www.epicweb.dev/talks/let-me-be

**Comment by vasco3** [comment]
I'm very curious about how Is this project monetized other than by the funding rounds.
